### PR TITLE
Default number is based on Usage Location and not Country

### PR DESCRIPTION
**Pull request by brianreidc7**
Default number is based on Usage Location and not Country. So have changed this. Tested a user - set the UsageLocation to IE and Country to blank and the Default number was USA. Assigned user a licence and they got an Irish phone number = UsageLocation

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/set-the-phone-numbers-included-on-invites-in-teams.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ A conferencing bridge gives you a set of dial-in phone numbers for your organiza
...
 
-For a new user, the default conferencing toll numbers is assigned based on the country that is set in the Office 365 profile of the user when the user is enabled for the Audio Conferencing service. If there is a toll number in the conference bridge that matches the country of the user, that number will be automatically assigned as the default toll number of the user. If there isn't one, the number that is defined as the default toll number of the conference bridge will be assigned as the default toll number of the user.  
+For a new user, the default conferencing toll numbers is assigned based on the Usage Location that is set in the Office 365 administration center of the user when the user is enabled for the Audio Conferencing service. If there is a toll number in the conference bridge that matches the country of the user, that number will be automatically assigned as the default toll number of the user. If there isn't one, the number that is defined as the default toll number of the conference bridge will be assigned as the default toll number of the user.
```
Comment (JohanFreelancer9) [inline Teams/set-the-phone-numbers-included-on-invites-in-teams.md]:
```suggestion
For a new user, the default conferencing toll numbers are assigned based on the Usage Location that is set in the Office 365 administration center of the user when the user is enabled for the Audio Conferencing service. If there is a toll number in the conference bridge that matches the country of the user, that number will be automatically assigned as the default toll number of the user. If there isn't one, the number that is defined as the default toll number of the conference bridge will be assigned as the default toll number of the user.  
```

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LolaJacobsen** [comment]
Ping'ing PM Neeraj Jain.

**Comment by LolaJacobsen** [comment]
Pinging Nakul Madaan.

**Comment by tonysmit** [comment]
This is correct now.

**Pull request merged**